apis/kueue/v1alpha1/finalizers.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+const (
+	ClusterQueueFinalizerName = "kueue.k8s.io/clusterqueue-in-use"
```

Comment:
Please move the other finalizer here too.

Although, maybe we can use exactly the same finalizer for both. They both have the same semantics: "in-use".